### PR TITLE
TCR -21 - mod-entities-links evaluation

### PR DESCRIPTION
**Pull request by vijay-eis**

### CODE REVIEW

File: module_evaluations/TCR-21_2022-12-22.MD [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Consider having a module documentation template that includes
+  - System requirements. Required version of JDK. Infrastructure and third party service integrations
+  - Overall design
+  - Data model
+  - Glossary
+  - Known Issues
+- Provide link to PERSONAL_DATA_DISCLOSURE in the template
+
+## Recommendations for the team
+- links/instances/{instanceId} --> links/instance-authority/{instanceId}
```
Comment (marcjohnson-kint) [inline module_evaluations/TCR-21_2022-12-22.MD:49]: What is the reasoning behind this recommendation?

File: module_evaluations/TCR-21_2022-12-22.MD [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- It will be useful to document system requirements for the module in the module documentation
+
+## Evaluation Notes
+- No third party system integrations
+- ~~PERSONAL_DATA_DISCLOSURE out of date and incomplete. Corrected by the team~~
+- Endpoint responds without tenant header
```
Comment (marcjohnson-kint) [inline module_evaluations/TCR-21_2022-12-22.MD:55]: What is meant by this?

Maybe this could be inline notes if it is relevant to a specific criteria.

Comment (craigmcnally) [reply module_evaluations/TCR-21_2022-12-22.MD]: I spoke with Vijay about this.  I think there was a misunderstanding here, caused by the wording of the criteria.

Comment (maccabeelevine) [reply module_evaluations/TCR-21_2022-12-22.MD:57]: If so, I shared the misunderstanding so please tweak the criteria.

**Comment by craigmcnally** [comment]
Formally approved by the Technical Council on 1/18/2023

**Pull request merged**